setup.cfg

```diff
@@ -10,5 +10,5 @@ max-line-length = 160
 test = pytest
 
 [tool:pytest]
-collect_ignore = ['setup.py']
+collect_ignore = ['setup.py','.generate-reports.py']
```

Comment:
Not here.

See my comment above.

Reply:
oh

Reply:
https://github.com/qbicsoftware/qube-cli/blob/master/setup.cfg#L5

I guess

Reply:
You can always try it locally before pushing by the way.

`pip install flake8`

Then just run `flake8` from the top level.